apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -24,6 +24,15 @@ export default class ColorPickerPropertyRow extends React.Component {
     window.removeEventListener('mousedown', this.handlePageClick);
   }
 
+  componentWillReceiveProps(nextProps) {
+    const {initialValue} = nextProps;
+    if (this.props.initialValue !== initialValue) {
+      this.setState({
+        value: initialValue
+      });
+    }
+  }
```

Comment:
Bump: In the [previous PR we talked about](https://github.com/code-dot-org/code-dot-org/pull/27644/files#r268336274) not depending on `state` at all for this anymore, wondering if that's still on the roadmap.